plugins/network-elements/nuage-vsp/src/net/nuage/vsp/acs/NuageVspPluginClientLoader.java

```diff
@@ -19,28 +19,86 @@
 
 package net.nuage.vsp.acs;
 
+import net.nuage.vsp.acs.client.NuageVspApiClient;
+import net.nuage.vsp.acs.client.NuageVspElementClient;
+import net.nuage.vsp.acs.client.NuageVspGuruClient;
+import net.nuage.vsp.acs.client.NuageVspManagerClient;
+import net.nuage.vsp.acs.client.NuageVspSyncClient;
+
 import java.net.MalformedURLException;
 import java.net.URL;
 import java.net.URLClassLoader;
 
 public class NuageVspPluginClientLoader {
 
-    private static NuageVspPluginClientLoader nuageVspPluginClientClassloader;
-    private ClassLoader loader = null;
+    private ClassLoader _loader = null;
+
+    private NuageVspApiClient _nuageVspApiClient;
+    private NuageVspElementClient _nuageVspElementClient;
+    private NuageVspGuruClient _nuageVspGuruClient;
+    private NuageVspManagerClient _nuageVspManagerClient;
+    private NuageVspSyncClient _nuageVspSyncClient;
+
+    public static final String NUAGE_PLUGIN_CLIENT_JAR_FILE = "/usr/share/nuagevsp/lib/nuage-vsp-acs-client.jar";
```

Comment:
Seems the location of the client jar should be configurable.  I am also concerned that we are included a plugin into the open source project that has an implicit requirement for a non open source library.  The expectation is that any plugin built in the open source release has all dependencies to run included in the build.  I will open a discussion on the mailing list to discuss further.